esp_encrypted_img/README.md

```diff
@@ -119,6 +119,10 @@ Note:
 
 For ESP32 devices that support the Digital Signature peripheral (such as ESP32-S2, ESP32-S3, ESP32-C3, etc.), you can enable hardware-accelerated RSA operations by selecting the `PRE_ENCRYPTED_RSA_USE_DS` configuration option. This provides enhanced security and performance compared to software-based RSA operations.
 
+**ESP-IDF Version Requirement:**
+
+The Digital Signature (DS) peripheral support was introduced in ESP-IDF v5.3. To use DS peripheral functionality with esp_encrypted_img, you must use ESP-IDF v5.3 or above. Projects using ESP-IDF versions prior to v5.3 cannot use the DS peripheral option.
```

Comment:
Does it mean decryption using DS peripheral? I think DS peripheral based sign support is there in IDF for quite some time..

Reply:
Yes, I have updated the doc to specifically mention DS support for decryption